src/sentry/web/frontend/integration_extension_configuration.py

```diff
@@ -57,8 +59,13 @@ def get(self, request, *args, **kwargs):
                 return self.redirect("/")
 
             # TODO(steve): we probably should check the user has permissions and show an error page if not
-            pipeline = self.init_pipeline(request, organization, request.GET.dict())
-            return pipeline.current_step()
+            try:
+                pipeline = self.init_pipeline(request, organization, request.GET.dict())
+                return pipeline.current_step()
+            except SignatureExpired:
+                return self.respond(
+                    "sentry/pipeline-error.html", {"error": "Installation link expired"},
```

Comment:
Is this a Django View?

Reply:
@NisanthanNanthakumar Yes. We use it for the integration installation pipeline.